src/measure_theory/function/conditional_expectation.lean

```diff
@@ -7,6 +7,7 @@ Authors: Rémy Degenne
 import analysis.inner_product_space.projection
 import measure_theory.function.l2_space
 import measure_theory.function.ae_eq_of_integral
```

Comment:
```suggestion
```
You could drop this now, as it is transitively imported via `radon_nikodym`.

(I'd encourage slight caution every time you add an import. In this case it doesn't delay the compilation of `conditional_expectation` by much, but it's always worth considering if instead of adding an import to an existing file, you should be creating a new file instead. Especially as measure_theory is already the slowest to compile part of mathlib, we don't want to "linearize" the import graph too much! :-)